assignments/src/cs/assignment01/RandomWalkers.java

```diff
@@ -0,0 +1,32 @@
+package cs.assignment01;
+
+public class RandomWalkers {
+    public static void main(String[] args) {
+        int n = Integer .parseInt(args[0]);
+        int t = Integer.parseInt(args[1]);
+        int x = 0;
+        int y = 0;
+        int i = (x * x) + (y * y);
+        while (t > 0)
+        {
+            t = t - 1;
+            while (n > 0)
+            {
+                double a = Math.random();
+                n = n - 1;
+                if (a <= 0.25) {
```

Comment:
Use a < 0.25, a < 0.5, a < 0.75, and a < 1 for testing because a is **less** than 1.